### ISSUE TITLE
Extension of the footer area in Greenlight v3 to include a link to an accessibility statement

### ISSUE THREAD

**Issue by GunnarPrahl** [bigbluebutton/greenlight#5686]
### Problem description

In the European Union public authorities and bodies governed by public law in all member states are required by law to include an accessibility statement on websites. If this accessibility statement is missing, citizens have the right to complain to the relevant authorities about the barriers on the website and, if possible, to demand their removal.

The legal basis for this obligation is Directive 2016/2102 of the European Union. The member states have transposed this into their own national laws. In Germany, for example, this is the "Behindertengleichstellungsgesetz". The Act on Equal Opportunities for People with Disabilities.

Article 7 of Directive 2016/2102 states:

"For websites, the accessibility statement shall be provided in an accessible format, using the model accessibility statement referred to in paragraph 2, and shall be published on the relevant website."

The original text of the Directive in English is published on the EUR-Lex portal, which publishes all legal sources of the European Union, see [https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32016L2102](url)

### Proposal for implementation in Greenlight v3:

Similar to the implementation for the legal notice and the privacy policy, a link should be included in the footer of all Greenlight pages. The URL for the link to the accessibility statement should be customizable in the basic settings of Greenlight, just like the URLs for the legal notice and the privacy policy.
